src/main/java/fi/otavanopisto/kuntaapi/server/rest/CodesApiImpl.java

```diff
@@ -0,0 +1,109 @@
+package fi.otavanopisto.kuntaapi.server.rest;
+
+import java.net.URI;
+import java.util.ArrayList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.ArrayList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)